src/main/webapp/js/build-pipeline.js

```diff
@@ -26,7 +26,7 @@ BuildPipeline.prototype = {
                         buildPipeline.updateAllBuildCards(dependencies);
                         // trigger all dependency tracking
                         jQuery.each(dependencies, function(){
-                            jQuery("#pipelines").trigger("show-status-" + this);
+                            jQuery("table.pipelines").trigger("show-status-" + this);

```

Comment:
This id vs class issue looks like a separate pre-existing issue that we can split to a new PR.

Reply:
Done in https://github.com/jenkinsci/build-pipeline-plugin/pull/158